mula/.ci/.env.test

```diff
@@ -1,22 +1,43 @@
-QUEUE_URI=amqp://guest:guest@ci_rabbitmq:5672/%2fkat
-
 POSTGRES_USER=postgres
 POSTGRES_PASSWORD=mysecretpassword
-POSTGRES_DB=scheduler
+DATABASE_MIGRATION=true
+
+QUEUE_URI=amqp://guest:guest@ci_rabbitmq:5672/%2fkat
+
+KATALOGUS_API=http://ci_katalogus:8000
+OCTOPOES_API=http://ci_octopoes:80
+BYTES_API=http://ci_bytes:8000
+SCHEDULER_API=http://ci_scheduler:8000
+BOEFJE_API=http://boefje:8000
 
 SCHEDULER_API_HOST="0.0.0.0"
 SCHEDULER_API_PORT="8000"
 SCHEDULER_PQ_MAXSIZE=0
 SCHEDULER_PQ_POPULATE_INTERVAL=60
 SCHEDULER_PQ_POPULATE_GRACE_PERIOD=1
-SCHEDULER_DB_USER=postgres
-SCHEDULER_DB_PASSWORD=mysecretpassword
+SCHEDULER_DB_USER=scheduler_app
+SCHEDULER_DB_PASSWORD=${POSTGRES_PASSWORD}
 SCHEDULER_DB=scheduler
 SCHEDULER_DB_URI=postgresql://${SCHEDULER_DB_USER}:${SCHEDULER_DB_PASSWORD}@ci_postgres:5432/${SCHEDULER_DB}
```

Comment:
Does this still work if `SCHEDULER_DB_USER` and `SCHEDULER_DB_PASSWORD` are removed?